api/src/main/java/org/hawkular/apm/api/model/events/CommunicationDetails.java

```diff
@@ -649,22 +674,22 @@ public void writeExternal(ObjectOutput oos) throws IOException {
      */
     public static class Outbound implements Externalizable {
 
-        private List<String> ids = new ArrayList<String>();
+        private List<String> linkIds = new ArrayList<String>();
```

Comment:
Yes, although where used in the event processing, I'm trying to avoid unnecessary creation of iterators - so use of lists just enables using index.

Will look again at this, as may not be necessary to have a collection.